jujugui/static/gui/src/app/jujulib/test-bakery.js

```diff
@@ -172,6 +179,10 @@ describe('Bakery', () => {
       bakery.discharge('macaroons', success, failure);
     });
 
+    // XXX j.c.sackett 2017-04-27 It would be better to test this using the
+    // public API, but all my attempts to put together a testing macaroon that
+    // works with all of this isn't working out for me. If you can fix it, plz
+    // do so.
```

Comment:
It's not a question of stubbing out macaroons; it's a question of assembling the right set of stubs &c that will allow calling to `discharge` to then properly exercise `_getThirdPartyDischarges`. I ceased having confidence that anything was being tested properly.

I'll just remove the comment, since I'm not convinced there is a better way.